grid.h

```diff
@@ -54,7 +54,13 @@ void initializeGrids(
 
     \param[in,out] mpiGrid The DCCRG grid with spatial cells
 */
-void balanceLoad(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid, SysBoundary& sysBoundaries);
+void balanceLoad(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid, SysBoundary& sysBoundaries,
+   bool doTranslationLists = true);
+
+/* helper for calculating AMR flags and cell lists and building pencils
+ */
+void prepareAMRLists(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid);
```

Comment:
Can go if not called outside grid.cpp?

Reply:
Is actually called in grid.cpp before the function is defined, so is needed.

Reply:
It can be forward declared in `grid.cpp` to reduce scope, but I'm not entirely sure either way is strictly better.